docs/wiki/99_Developers/Developers_Guide.md

```diff
@@ -0,0 +1,215 @@
+
+### PR Formatting
+Be sure to observe and follow team conventions for how PRs should be named and described. 
+
+**Title:** [module_name] Quick description   
+
+**[Contents](../../../.github/PULL_REQUEST_TEMPLATE.md) of the initial comment:** Include a brief summary of the changes and rationale, links to related issues or PRs, and testing instructions if there are any. 
+
+**Linking issues:** Use GitHub keywords like `Fixes` or `Resolves` so that related issues are closed automatically when the PR is merged. Do not use these keywords if you are only addressing one point raised in an issue. This will close the entire issue and leave unresolved bugs in the codebase.
+**Note** that automatic issue-linking and issue-closing only works on the default branch.  If the PR is not on the default branch, you will need to manually close the issue after the PR is merged. 
+
+**Labels:** On the right side of the browser, you will see the option to add labels. Please add any that are related to the PR. 
+**If** you are working on release testing, make sure to include the label (ex. 23.0.0-testing) to the PR.
+
+**Reviewers/Assignees:** Reviewers are people that you would like to ping specifically, but are not the same as assignees.
+Be sure to consult the Contributing and Code Review guidelines. For example, 2 reviewers must approve the PR and any code change must be pulled and tested by a developer before a PR is marked as `Passed Manual Tests`. 
+
+**Travis:** The Travis build status and results are linked near the bottom of your PR in GitHub. 
+You can restart Travis if it gives error messages that seem inappropriate, or if the build stalls (for longer than 1h).
+if you are pushing a last typo fix on a PR, it is okay to add `[skip ci]` in the commit message. It is not okay to do this if it's the first commit or if you are fixing more than documentation typos.
+
+**Documentation PRs:** If working specifically on markdown files or other documentation, be sure to check the actual look of the document once you have pushed your PR and fix any formatting errors. You can do this by going to “Files Changed” and viewing how your markdown file will actually appear in GitHub. Don’t forget to click all links to make sure they won't be broken when the document is rendered. 
+
+## 4. Code Review
+
+### Testing or Contributing to someone's PR
+First, make a remote of their fork:
+
+    git remote add "their_fork" <their_fork_url>
+          
+Then, fetch their remote fork and checkout the branch related to their PR:
+
+    git fetch their_fork
+    git checkout their_fork/branch_name
+
+Alternatively, to avoid being under the detached HEAD state, checkout and create a local branch with the command
+
+    git checkout -b branch their_fork/branch_name
+
+Follow the provided testing instructions to test their code on your VM. 
+
+If you made changes (with their permission), commit and push them to that branch:
```

Comment:
Now we need to request permission to be able to push on someone else repository. Maybe that worth a mention?

Reply:
That's a good point. Altho this is for external devs so I'm not quite sure if its the same process for them... 
@christinerogers Do you think more needs to be said here?